lib/licensee/project.rb

```diff
@@ -2,17 +2,6 @@ class Licensee
   class Project
     attr_reader :repository
 
-    # Array of file names to look for potential license files, in order
-    # Filenames should be lower case as candidates are downcased before comparison
-    LICENSE_FILENAMES = %w[
```

Comment:
You did and we talked about it, but I believe we reached a conclusion that it doesn't matter how regexes are read, because files are tested one by one in sequence. "COPYING" will match the regex and get assigned a score of "1", then LICENSE will match and also get assigned a score of "1", and so on.